src/60_Samples_%26_Templates/Product_Page.html

```diff
@@ -290,6 +306,74 @@
       z-index: 1;
       position: absolute;
       right: 0;
+      text-transform: uppercase;
+      margin: var(--space-3);
+    }
+
+    .drawermenu a {
+      text-transform: uppercase;
+      text-decoration: none;
+      display: block;
+      padding: var(--space-3);
+    }
+
+    .watch-video {
+      text-transform: uppercase;
+      margin-bottom: var(--space-4);
+    }
+
+    .order {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .color-selector, .quantity-selector, .size-selector, .submit-selector, .related-products-container {
+      display: flex;
+      -webkit-box-align:center;
+      -ms-flex-align:center;
+      align-items:center
+    }
+
+    .color-selector ul, .quantity-selector ul, .size-selector ul {
+      padding: 0;
+      margin: var(--space-2);
+    }
+
+    .related-products {
+      display: inline-block;
+    }
+
+    .margin1, .related-products {
```

Comment:
Can you find an approach that doesn't require encoding styles into classes?

Reply:
For example: you can wrap the sample in a div `<div class="product-sample">` and then use `.product-sample > * { margin: ... }` to assign all children a margin.